config/boards/bananapif3.conf

```diff
@@ -0,0 +1,30 @@
+# SpacemiT K1 octa core RISC-V SoC 2GB/4GB RAM 8GB/16GB eMMC 4x USB3 2x GbE
+BOARD_NAME="Banana Pi F3"
+BOARDFAMILY="spacemit"
+BOARD_MAINTAINER=""
```

Comment:
is this supposed to be blank?

Reply:
I don't plan to be the official maintainer, so I left it empty. Do you want the job?

Reply:
Having it empty is correct -- Igor's secret script will PR updates to it later. Not-having it will block such machinery from working

Reply:
If the board does not have a maintainer attached, shouldn't it be .csc then?